wordpress-scripts-webpack.config.js

```diff
@@ -1,6 +1,7 @@
 /**
  * External Dependencies
  */
+const fs = require('fs');
```

Comment:
Do we still need this?

Reply:
5f63fe0157fc24a20547aabf7eb571dd2f5d9d80